TestDB.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import sys; sys.path.append('../')
+import faker
+
+class TestUser(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
+        self.faker = faker.Faker()
+        faker = self.faker
+
+    def test_save_customer(self):
+        customer = Customer()
+        customer.first_name = faker.first_name()
+        customer.last_name = faker.last_name()
+        customer.date_created = faker.date()
+        user = User()
+        customer = user.save(customer)
```

Comment:
Explain the reasoning for needing an `User` instance in order to add a customer to the database. While #4 outlines the desired User Experience, it is not necessary to codify that at this level. You will create code for the UI that will ensure that users are creating customers.

Note: Going down this path will lead you to answer -> How can you generalize the `save` method given the User will be creating all sorts of items?

Can the model classes you're creating `Customer`, `User` be in charge of their own "saving"?